File: Bedu/Java/Clase6/ShareScroll/app/src/main/java/com/charlye934/sharescroll/MainActivity.java
package com.charlye934.sharescroll;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void compartir(View view) {
        //Recurso que vamos a compartir
        Intent sendIntent = new Intent();
        sendIntent.setAction(Intent.ACTION_SEND);
        sendIntent.putExtra(Intent.EXTRA_TEXT, "Texto a compartir");
        sendIntent.setType("text/plain");

        //Por donde lo vamos a compartir
        Intent shareIntent = Intent.createChooser(sendIntent,null);
        startActivity(shareIntent);
    }
}

File: Bedu/Kotlin/Clase2-3/ContenidoDinamico/app/src/main/java/com/charlye934/contenidodinamico/MainActivity.kt
package com.charlye934.contenidodinamico

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.*
import android.widget.LinearLayout.LayoutParams
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var cont:Int = 1
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun generarItem(view: View) {
        var empresa = TextView(this)

        val params = LinearLayout.LayoutParams(LayoutParams.MATCH_PARENT,LayoutParams.MATCH_PARENT)
        empresa.layoutParams = params
        empresa.text = "Empresa 2"
        contenedor.addView(empresa)

    }
}
